editor/tasks/leiningen/pack.clj

```diff
@@ -29,14 +29,10 @@
                     "lib" ["libparticle_shared.dylib"]}
    "arm64-macos" {"bin" ["dmengine"]
                   "lib" ["libparticle_shared.dylib"]}
-   "x86-win32"     {"bin" ["dmengine.exe" "dmengine.pdb"]
-                    "lib" []}
    "x86_64-win32"  {"bin" ["dmengine.exe" "dmengine.pdb"]
                     "lib" ["particle_shared.dll"]}
    "x86_64-linux"  {"bin" ["dmengine"]
-                    "lib" ["libparticle_shared.so"]}
-   "arm64-ios"  {"bin" ["dmengine"]
-                    "lib" []}})
```

Comment:
I found this comment: 
<img width="647" alt="image" src="https://github.com/defold/defold/assets/2209596/3eb49698-aa02-4bcf-a716-d3b9e6a2a4ec">

This function removed a long time ago. So we don't need ios dmengine in this folder anymore